content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -521,3 +519,19 @@ Interactive transaction error:
 - Could not parse interactive transaction ID
 - Could not find Query Engine for the specified host and transaction ID
 - Could not start interactive transaction
+
+### Accelerate
+
+As Accelerate connections operate differently from local database connections, error codes are different from regular ones. In general, all Accelerate-related errors start with `P6xxx`.
```

Comment:
They are not different as far as I know, there are only some additional ones that are triggered by Accelerate only

Reply:
Updating copy to

> Accelerate-related errors start with `P6xxx`.